schemas/SvrPropertiesInDeviceState.json

```diff
@@ -0,0 +1,103 @@
+{
+"key":{
+    "R": "Read Only",
+    "RW": "Read Write",
+    "-": "No Access",
+    "": "Unspecified"
+},
+"doxm":{
+    "DOXS":{
```

Comment:
/doxs matches IoTivity a7cb85abe19589bb1f7d5854dc18644c642766cc... looks ok to me.